DALI.tex

```diff
@@ -800,6 +816,36 @@ \subsubsection{Circle}
 of a table), but specific services may define something that is applicable in a
 more limited context.
 
+\subsubsection{Range}
```

Comment:
hmmm, SIA and SODA also do not discuss this. with the text as written, I would interpret that
`359 1 -1 1` spans the meridian and is small (2x2 deg) and technically `1 359 -1 1` would be a large band 358 deg x 2 deg... that's what you meant by "reversed longitude"? I've rewritten it this way.

that probably means I have to be careful with using the word "interval" since 359,1 would not be a valid interval (it wasn't intended to be, but I'll fix that anyway).

The same thing kind of comes up in polygons, where they look pretty odd to the eye in this case

Reply:
Thanks, I think the rewrite is OK.

With reference to polygons, [STC 1.33 sec 4.5.1.4](https://www.ivoa.net/documents/REC/DM/STC-20071030.html#_Toc181531824) resolves this by saying
> In order to avoid ambiguities in direction, vertices need to be less than 180° apart in both coordinates.

Should we explicitly defer to STC concerning polygons?  But this is probably wider than DALI - see [Alberto's recent Interop talk](https://wiki.ivoa.net/internal/IVOA/InterOpMay2023Ops/ESO_VO_Polygon_Tool.pdf).

Reply:
The intent was that we were referring to the STC definition in it's entirety, but the text was too brief and gave a different impression.  That's more or less a bug... erratum?

Now that STC-1.33 is really old and more or less obsoleted by Coords and Meas, I wonder if we should fork the complete definition and include it in DALI.

Reply:
> Now that STC-1.33 is really old and more or less obsoleted by Coords and Meas, I wonder if we should fork the complete definition and include it in DALI.

You'd have my vote for it.

For Range (if we want it), I think the only sane convention is to let RA be in (-360,360] or in [0, 720] or something comparable.  Otherwise, you'll get insane trying to define the behaviour along the stiching line.